exercises/protein-translation/example.js

```diff
@@ -0,0 +1,63 @@
+const getProtein = (codon) => {
+  switch (codon) {
+    case 'AUG':
+      return 'Methionine';
+
+    case 'UUU':
+    case 'UUC':
+      return 'Phenylalanine';
+
+    case 'UUA':
+    case 'UUG':
+      return 'Leucine';
+
+    case 'UCU':
+    case 'UCC':
+    case 'UCA':
+    case 'UCG':
+      return 'Serine';
+
+    case 'UAU':
+    case 'UAC':
+      return 'Tyrosine';
+
+    case 'UGU':
+    case 'UGC':
+      return 'Cysteine';
+
+    case 'UGG':
+      return 'Tryptophan';
+
+    case 'UAA':
+    case 'UAG':
+    case 'UGA':
+      return 'STOP';
+
+    default:
+      return 'INVALID';
+  }
+};
```

Comment:
Nothing at all wrong with the massive `switch` statement, but I sometimes like an alternative approach to these sorts of things, using an object.  EG:

```javascript
const ACID_PROTEIN_MAP = {
  'AUG': 'Methionine',
  'UUU': 'Phenylalanine',
  'UUC': 'Phenylalanine',
// and so on...
};

const getProtein = codon => ACID_PROTEIN_MAP[codon] || 'INVALID';
```

It remains very readable and gets what's essentially configuration out of your logic.  In production, you might pull the config out even further: this makes it easy to adjust the mapping without having to redeploy code.